azure-function/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "ms-azuretools.vscode-azurefunctions"
+    ]
+}
```

Comment:
This doc can be deleted since all extensions are captured at the project root level